### PR TITLE
Meta morpho scripts

### PR DESCRIPTION
**Pull request by tbrent**

### CODE REVIEW

File: scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  const collateral = <MetaMorphoFiatCollateral>await MetaMorphoFiatCollateralFactory.connect(
+    deployer
+  ).deploy(
+    {
+      priceTimeout: PRICE_TIMEOUT.toString(),
+      chainlinkFeed: USDT_USD_FEED,
+      oracleError: USDT_ORACLE_ERROR.toString(),
+      erc20: networkConfig[chainId].tokens.bbUSDT,
+      maxTradeVolume: fp('1e6').toString(), // $7.5m vault
```
Comment (akshatmittal) [inline scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts:63]: I think we should be using _much_ smaller numbers for this. The vaults are currently too small to sustain these.

**Review by akshatmittal — CHANGES_REQUESTED** [review]

File: scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts (hunk 1 above)
Comment (tbrent) [reply scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts:63]: Ok I think $500k can make sense, but much lower and things become unusable on mainnet

Comment (tbrent) [reply scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts:63]: And <$500k for pyUSD, since it has $1.7m vault

Comment (akshatmittal) [reply scripts/deployment/phase2-assets/collaterals/deploy_bbusdt.ts]: That sounds good to me. Probably $250k for smaller vaults and $500k for every other.

**Review by akshatmittal — APPROVED** [review]

**Pull request merged**